packages/services/examples/typescript-browser-with-output/src/index.ts

```diff
@@ -16,25 +17,20 @@ import {
 
 import { Kernel } from '@jupyterlab/services';
 
-function main() {
+async function main() {
   const code = [
-    'import numpy as np',
-    'import matplotlib.pyplot as plt',
-    '%matplotlib inline',
-    'x = np.linspace(-10,10)',
-    'y = x**2',
-    'print(x)',
-    'print(y)',
-    'plt.plot(x, y)'
+    'from IPython.display import HTML',
+    'HTML("<h1>Hello, world!</h1>")'
   ].join('\n');
   const model = new OutputAreaModel();
   const rendermime = new RenderMimeRegistry({ initialFactories });
   const outputArea = new OutputArea({ model, rendermime });
 
-  void Kernel.startNew().then(kernel => {
-    outputArea.future = kernel.requestExecute({ code });
-    document.getElementById('outputarea').appendChild(outputArea.node);
-  });
+  const kernel = await Kernel.startNew();
+  outputArea.future = kernel.requestExecute({ code });
+  document.getElementById('outputarea').appendChild(outputArea.node);
+  await outputArea.future.done;
+  console.log('Test complete!');
```

Comment:
Thanks for this @blink1073 :). Really helps me jump back into programming with @jupyterlab tools with the new changes :).

Thank you!

Reply:
No problem @SimonBiggs, nice to e-see you again!

Reply:
:). Also, this does remind me, I do have quite a bit else to thank you for. I learnt so much off of all the work you've done here. For example I have used what I learnt about Lerna here to implement a Python monorepo https://github.com/pymedphys/pymedphys/tree/master/packages. I couldn't have done it without your example, so thank you :).

I also got the ideas of dependency graphing off of you guys:

https://pymedphys.com/developer/dependencies.html

And many, many other things. So, thank you. :) Thanks @blink1073 :).

Reply:
My pleasure.  One day we'll meet and you can buy me a beer.  ;)

Reply:
Now that, is a good plan :)